src/Avalonia.Controls/NumericUpDown/NumericUpDown.cs

```diff
@@ -641,6 +638,9 @@ protected virtual void OnSpin(SpinEventArgs e)
             {
                 DoDecrement();
             }
+
+            var handler = Spinned;
+            handler?.Invoke(this, e);
```

Comment:
nit: Since you are changing this code it would be good to remove the extra `handler` variable. Just `Spinned?.Invoke(...)` is convention.